addons/traefik-forward-auth/1.0.x/traefik-forward-auth-4.yaml

```
@@ -0,0 +1,92 @@
+---
+apiVersion: kubeaddons.mesosphere.io/v1beta1
+kind: Addon
+metadata:
+  name: traefik-forward-auth
+  namespace: kubeaddons
+  annotations:
+    catalog.kubeaddons.mesosphere.io/addon-revision: "1.0.4-4"
```

We might as well add the appversion here.

```suggestion
    catalog.kubeaddons.mesosphere.io/addon-revision: "1.0.4-4"
    appversion.kubeaddons.mesosphere.io/opsportal: "1.0.4"
```